### PR TITLE
Commando18 commerce 4866

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-test/src/testIntegration/java/com/liferay/commerce/product/service/test/CPDefinitionLocalServiceTest.java [hunk 1]
```diff
@@ -386,6 +387,35 @@ public void testDeleteCPDefinitionWithIgnoreSKUCombinationsAndDefaultInstance()
...
 	}
 
+	@Test
+	public void testUpdateCPDefinitionExternalReferenceCode() throws Exception {
+		frutillaRule.scenario(
+			"Update product definition external reference code"
+		).given(
+			"I add a product definition"
+		).when(
+			"externa reference code is set"
```
Comment (dsitu) [inline modules/apps/commerce/commerce-product-test/src/testIntegration/java/com/liferay/commerce/product/service/test/CPDefinitionLocalServiceTest.java:397]: typo
```suggestion
			"external reference code is set"
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Pull request closed without merging**